<launch>
 	<node pkg="starleth_elevation_mapping" type="starleth_elevation_mapping" name="starleth_elevation_mapping" output="screen">
    	 <param name="point_cloud_topic" value="/depth_registered/points_throttled" />
    	 <param name="map_frame_id" value="/map" />
    	 <param name="track_point_frame_id" value="/map" />
    	 <rosparam command="load" file="$(find starleth_elevation_mapping)/parameters/terrain_scan.yaml" />
    	 <rosparam command="load" file="$(find starleth_elevation_mapping)/parameters/sensors/kinect_nguyen_et_al.yaml" />
    </node>
</launch>